packages/menu/src/use-menu.ts

```diff
@@ -162,13 +169,13 @@ export function useMenu(props: UseMenuProps = {}) {
     })
   }, [])
 
-  const focusFirstItem = React.useCallback(() => {
+  const focusInitialItem = React.useCallback(() => {
     const id = setTimeout(() => {
-      const first = descendants.firstEnabled()
-      if (first) setFocusedIndex(first.index)
+      const itemIndex = initialFocusIndex ?? descendants.firstEnabled()?.index
```

Comment:
the type has changed from `descendants.firstEnabled()` to `descendants.firstEnabled()?.index`